frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -595,13 +595,21 @@ function ReaderHighlight:onPanelZoom(arg, ges)
         }
         UIManager:show(imgviewer)
     end
+    logger.dbg("File:", self.ui.document.file)
+    logger.dbg("Extension:", util.getFileNameSuffix(self.ui.document.file))
     return true
 end
 
 function ReaderHighlight:onHold(arg, ges)
+    -- if it's manga/comic, panel zoom
+    local filetype = util.getFileNameSuffix(self.ui.document.file)
+    if filetype == 'cbz' or filetype == 'djvu' then
```

Comment:
Nit:
```suggestion
    if filetype == "cbz" or filetype == "djvu" then
```

This is just a proof of concept atm I assume? Otherwise I might have bigger comments. :-P

Reply:
It's PoC for now, I'll try to get it ready for review tomorrow, when I add menu entry to enable/disable this feature (and figure out where to put it ;-))